agr_java_core/src/main/java/org/alliancegenome/api/entity/DiseaseAnnotationDocument.java

```diff
@@ -22,13 +22,13 @@
 public class DiseaseAnnotationDocument extends SearchableItemDocument {
 
 	private String uniqueId;
-	private VocabularyTerm diseaseRelation;
+	private VocabularyTerm relation;
 	private DOTerm object;
 	private List<ECOTerm> evidenceCodes;
 	private Reference singleReference;
 	private Set<Reference> references;
 
-	private String diseaseRelationNegation;
+	private String generatedRelationString;
```

Comment:
@oblodgett @markquintontulloch Better like this?

Reply:
Yes thanks... this makes a lot more sense to me.